packages/hsl-shared-components/src/TextInput/TextInputNative.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components/native';
-import omit from 'lodash/fp/omit';
+import omit from 'lodash/omit';
 import { size } from '../utils';
 
 const Input = styled.TextInput`
```

Comment:
I noticed the input in iOS is collapsed and looks like in the image above.
Small dots are text.

One fix is to add `height: ${size(60)};` to the native input styles.

<img width="399" alt="screen shot 2017-10-10 at 12 06 34" src="https://user-images.githubusercontent.com/3703618/31378394-c94160f4-adb3-11e7-9384-26e2247eddce.png">